storage/innobase/lock/lock0lock.cc

```diff
@@ -7110,13 +7110,32 @@ and less modified rows. Bit 0 is used to prefer orig_trx in case of a tie.
       lock_cancel_waiting_and_release<true>(victim->lock.wait_lock);
```

Comment:
@temeo This call will release locks that victim is waiting and it was already selected as deadlock victim.  I will investigate can we remove actual rollback handling from here.